src/pe/header.rs

```diff
@@ -543,6 +543,7 @@ pub struct Header {
 }
 
 impl Header {
+    /// Parses PE header from the given bytes
```

Comment:
thank you for adding documentation here; can you add the part about how we will fail if dos stub or header is wrong?

e.g.:

```
Parses PE header from the given bytes; this will fail if the DosHeader or DosStub is malformed or missing in some way
```